documentation/docs/25-build-and-deploy/90-adapter-vercel.md

```diff
@@ -63,9 +63,9 @@ export const config = {
 
 The following options apply to all functions:
 
-- `runtime` can be `'edge'`, `'nodejs16.x'` or `'nodejs18.x'` (the default)
-- `regions` can be `'all'` (the default if `runtime` is `'edge'`) or an array of [edge network regions](https://vercel.com/docs/concepts/edge-network/regions) (defaulting to `["iad1"]` for serverless functions)
-- `split` will, if `true`, cause a route to be deployed as an individual function. If `defaultConfig.split` is `true`, all routes will be deployed as individual functions
+- `runtime`: `'edge'`, `'nodejs16.x'` or `'nodejs18.x'` (the default)
+- `regions`: an array of [edge network regions](https://vercel.com/docs/concepts/edge-network/regions) (defaulting to `["iad1"]` for serverless functions) or `'all'` if `runtime` is `edge` (its default)
```

Comment:
we should check, but I _think_ you can have `all` even with lambda deployments

Reply:
I infered this from reading https://vercel.com/docs/build-output-api/v3#vercel-primitives/serverless-functions/configuration where it's typed as `string[]` only